tests/xharness/Jenkins.cs

```diff
@@ -219,8 +224,19 @@ bool IsIncluded (TestProject project)
 		{
 			if (!project.IsExecutableProject)
 				return false;
+			
+			if (project.IsBclTest) {
+				if (!project.IsNewBclTest)
+					return IncludeBcl || IncludeOldBCL;
+				if (project.IsMscorlib) 
+					return IncludeMscorlib;
+				return IncludeBcl || IncludeNewBCL;
+			}
 
-			if (!IncludeBcl && project.IsBclTest)
+			if (!IncludeMonotouch && project.IsMonotouch)
+				return false;
+
+			if (!IncludeXamarin && !project.IsBclTest && !project.IsMonotouch)
```

Comment:
`!project.IsBclTest` is redundant, because `IsBclTest` will always be false at this point.

Reply:
I don't like the name `Xamarin` for all tests except monotouch-test, it's very unintuitive. What about something more awkward, but understandable, like `NonMonoTouch`, so you'd end up with `IncludeNonMonotouch`, `IsNonMonotouch`, etc?